NetworkingSampleApp/NetworkingSampleApp/Scenes/Download/DownloadProgressViewModel.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  DownloadRowViewModel.swift
+//  NetworkingSampleApp
+//
+//  Created by Matej Molnár on 07.03.2023.
+//
+
+import SwiftUI
```

Comment:
```suggestion
import Foundation
```

Btw. is it becoming a thing to import UI framework into the view model nowadays?

Reply:
Nice catch, please no UI related stuff on VMs, maybe some exceptions in UIKit(eg I saw VM as diffableDataSource provider and that we can overcome 💁‍♂️)